incremental-copy-docs/README.md

```diff
@@ -76,6 +93,8 @@ To quickly deploy the Migration tool, you can use the Azure deployment below. Pl
 				- <*path-to-template*>: Provide the path to your ARM/Bicep template file i.e. main.json under infra folder.
 				- <*path-to-parameter*>: Specify the path to the parameters file i.e. armmain.parameters.json under infra folder.
 
+      **NOTE** : To avoid azure resource already deployed with the same name use unique Prefix for FHIR Migration Tool resources during deployment.
```

Comment:
Capitalization: "azure" should be "Azure".

Reply:
Done with the changes.